config/initializers/assets.rb

```diff
@@ -6,7 +6,7 @@
 Rails.application.config.assets.version = '1.0'
 
 # Add additional assets to the asset load path.
-# Rails.application.config.assets.paths << Emoji.images_path
+Rails.application.config.assets.paths << Rails.root.join('node_modules')
```

Comment:
There might be alternate way to do this via webpacker more directly instead of asset pipeline?

Reply:
I do not think it is a good idea to have the whole `node_modules/` directory processed by the assets pipeline

Reply:
Agreed ... updated this to use an approach that pulls from configured webpack assets instead of sprockets/rails asset pipeline.